modules/product-alerts-magento-px/services/useCustomerNotifyInStock.ts

```diff
@@ -0,0 +1,9 @@
+import useCustomerNotifyInStockRepository from '#ioc/repositories/useCustomerNotifyInStockRepository'
+
+export default () => {
+  const customerNotifyInStockRepository = useCustomerNotifyInStockRepository()
+
+  return async (...args: Parameters<typeof customerNotifyInStockRepository>) => {
```

Comment:
Do not use this hack for parameters anymore. Specify parameters here properly. I know its a bit of redundancy for now but its cleaner design.